libs/core/src/lib/calendar/calendar-header/calendar-header.component.ts

```diff
@@ -212,6 +218,23 @@ export class CalendarHeaderComponent<D> implements OnDestroy, OnInit, OnChanges
     /** Get information about amount of years displayed at once on year view  */
     private _amountOfYearsPerPeriod = 1;
 
+    /**
+     * Function used to disable previous button in the calendar header.
+     */
+
+    @Input()
+    set previousButtonDisableFunction(disableFunc: (date?: D) => boolean) {
+        this.previousButtonDisabled = disableFunc();
```

Comment:
Current date not passed to the function, but i guess this is the idea of this feature. Also the function should be invoked every time new date is selected, not only once when binding.